### PR TITLE
UI refactor

### PR DESCRIPTION
**Pull request by wxxedu**
Now the UI should be working. To make it work

### CODE REVIEW

File: src/main/java/seedu/address/MainApp.java [hunk 1]
```diff
@@ -85,7 +85,7 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         ReadOnlyAddressBook addressBook;
         try {
             addressBookOptional = storage.readAddressBook();
-            if (!addressBookOptional.isPresent()) {
+            if (addressBookOptional.isEmpty()) {
```
Comment (BoAi01) [inline src/main/java/seedu/address/MainApp.java:88]: Lol, this is umm.. subtle. : )

File: src/main/java/seedu/address/MainApp.java [hunk 2]
```diff
@@ -172,7 +174,7 @@ protected UserPrefs initPrefs(UserPrefsStorage storage) {
             initializedPrefs = prefsOptional.orElse(new UserPrefs());
         } catch (DataConversionException e) {
             logger.warning("UserPrefs file at " + prefsFilePath + " is not in the correct format. "
-                                   + "Using default user prefs");
+                               + "Using default user prefs");
             initializedPrefs = new UserPrefs();
```
Comment (BoAi01) [inline src/main/java/seedu/address/MainApp.java:178]: Thanks for correcting the indentation

**Review by BoAi01 — APPROVED** [review]
TODO: add javadoc and more test cases later on.

we need to move on now. merged.

File: src/main/java/seedu/address/MainApp.java (hunk 1 above)
Comment (wxxedu) [reply src/main/java/seedu/address/MainApp.java:88]: The IDE gives me some lints, so i just corrected it according to the IDE

**Pull request merged**